guides/deploying-with-kubernetes.md

```diff
@@ -0,0 +1,288 @@
+## At a Glance
+
+Now that you've added your customizations, made your front-end look and perform great, what's next. Well it's time 
+to deploy your site to the cloud. This guide walks you through the process of deploying your site using [Kubernetes]
+(https://kubernetes.io/) Google's technology for orchestrating Docker containers at scale.
+
+## What you need
+
+**Prerequisites**
+
+As this document is focused upon deploying MailChimp Open Commerce, using Helm, some working knowledge of your chosen cloud provider and Kubernetes is required. For example, you will need a cloud hosted Kubernetes environment, and relevant administration/deployment permissions to said environment, before being able to proceed with the below. You will also need **kubectl** installed and configured to proceed.
+
+This documentation has been written for and tested on Linux specifically, but all the included tools and components are also available for other operating systems such as Windows and Mac. Please consult each components/tools bespoke documentation for guidance regarding non-Linux installation and configuration.
```

Comment:
Any point asking for a PR to these docs here if someone on a different OS finds a different step or tweak to a step is needed?